runc/plan.sh

```diff
@@ -1,12 +1,44 @@
 pkg_name="runc"
 pkg_origin="core"
-pkg_version=0.1.1
+pkg_version=1.0.0-rc10
 pkg_description="CLI tool for spawning and running containers according to the OCI specification"
 pkg_upstream_url="https://www.opencontainers.org/"
 pkg_license=('Apache-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source="https://github.com/opencontainers/runc"
-pkg_shasum="73549dc3eb29005dae5248385c6b40310a323b817714853d35d7194abfa357b8"
 pkg_bin_dirs=(bin)
 pkg_scaffolding=core/scaffolding-go
-pkg_build_deps=(core/pkg-config core/libseccomp)
+pkg_build_deps=(core/pkg-config
+                core/libseccomp)
+
+# We have to override a handful of callbacks from the go scaffolding
+# in order for things to work properly.
```

Comment:
I just took a peak at the source and it appears runc now uses go modules.  I think for packages that use go modules we might want to stop using the scaffolding completely. It really provides little benefit other than obfuscating the build.

Reply:
Good call; I'll see about removing the scaffolding altogether.

Reply:
I took a poke at this and it appears that the `go modules` support is currently on master but not yet part of a tag.  Let's leave it for now and make this transition on the next version bump.